src/SmartCommander/ViewModels/OptionsViewModel.cs

```diff
@@ -51,10 +51,11 @@ public OptionsViewModel()
             ConfirmationWhenDeleteNonEmpty = Model.ConfirmationWhenDeleteNonEmpty;
             SaveWindowPositionSize = Model.SaveWindowPositionSize;
             IsDarkThemeEnabled = Model.IsDarkThemeEnabled;
+            AllowOnlyOneInstance=Model.AllowOnlyOneInstance;
```

Comment:
whitespaces are missing before and after =

Reply:
fixed